tests/Resources/keywords_sidlistningsblock.robot

```diff
@@ -8,8 +8,7 @@ Sidlistningsblock Test
 
     # [Documentation] Verifies that block is existing and isnt empty
 
-    Wait Until Element Is Visible               css:#main-content > div.row.content-area-wrapper > div > div > div
-    Wait Until Element Is Visible               css:#main-content > div.row.content-area-wrapper > div > div > div > ul > li:nth-child(1)
-    ${elementsinblocks} =                       Get Element Count    css:[class="block__row media "]
+    Wait Until Element Is Visible               css:.pagelistblock
+    ${elementsinblocks} =                       Get Element Count    css:.pagelistblock
```

Comment:
Frågan är om vi inte bara testar samma sak två gånger här... Först kollar vi om vi hittar blocket. Sen räknar vi antalet block och kontrollerar att de är fler än 0. DVS om vi hittar blocket. Frågan är om vi inte skulle kunna köra `Wait Until Element Is Visible` på `.pagelistblock li a` typ..?

Reply:
Helt sant. Jag testar det.